### PR TITLE
bot to change lbry:// urls to open.lbry.io[done]

### PR DESCRIPTION
**Pull request by MSFTserver**
image

### CODE REVIEW

File: bot/modules/lbrylink.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    var text = msg.content.replace("lbry://", "https://open.lbry.io/");
+    var message = GetWordByPos(text, link)
+    var name = message.replace("https://open.lbry.io/", "");
+    const embed = {
+					"description": msg.author+", I see you tried to post a LBRY URL, here's a friendly hyperlink to share and for others to access your content with a single click: \n"+"[lbry://"+name+"]("+message+")",
+					"color": 7976557,
+					"author": {
+					     "name": "LBRY Linker",
+					     "icon_url": "https://i.imgur.com/yWf5USu.png"
+							}
```
Comment (aayanl) [inline bot/modules/lbrylink.js]: out of place }

File: bot/modules/lbrylink.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+					};
+    msg.channel.send({embed})
+  }
+})
+function GetWordByPos(str, pos) {
```
Comment (aayanl) [inline bot/modules/lbrylink.js]: formatting

**Comment by MSFTserver** [comment]
image

**Comment by MSFTserver** [comment]
https://github.com/lbryio/lbry-wunderbot/issues/87

**Pull request merged**